addons/website/views/website_templates.xml

```diff
@@ -769,14 +795,11 @@
                     <div class="row align-items-center">
                         <div class="col-lg-8">
                             <small>
-                                <span class="text-muted"><b>Contact us</b></span>
-                                <i class="fa fa-1x fa-fw fa-phone ml-3 mr-2"/><a href="tel:+1 (650) 555-0111">+1 (650) 555-0111</a>
                                 <i class="fa fa-1x fa-fw fa-envelope ml-3 mr-2"/><span><a href="mailto:info@yourcompany.example.com">info@yourcompany.example.com</a></span>
                             </small>
                         </div>
                         <div class="col-lg-4 text-lg-right">
                             <div class="s_share no_icon_color" data-name="Social Media">
-                                <small class="s_share_title text-muted"><b>Follow us</b></small>
```

Comment:
Did you grep the s_share_title class before removing part of a snippet? With this, the "Title position" is not there anymore.

Reply:
You looked at the option I mentioned or you just answered without searching?

Reply:
i did search before answering